### PR TITLE
Implemented a more reliable means to tint the Toolbar Overflow button

### PR DESCRIPTION
**Pull request by timusus**
Previously, the overflow icon was found by searching the toolbars for views matching the overflow’s content description.

This proved unreliable.. Hard to know exactly why, but in some cases findViewsWithText() was failing to find the overflow image view, so it didn’t get themed correctly.

This change just retrieves the overflow icon via toolbar.getOverflowIcon(), tints it, then sets it again via toolbar.setOverflowIcon().

### CODE REVIEW

**Comment by afollestad** [comment]
It looks like your commit before I merged `master` failed on Travis, it's most likely due to code formatting

**Comment by timusus** [comment]
No probs, I'll check it out.

**Comment by timusus** [comment]
If my interpretation of the Travis logs is correct:

> /home/travis/build/afollestad/aesthetic/sample/src/main/java/com/afollestad/aestheticsample/MainActivity.java uses or overrides a deprecated API.

The issue appears to be the `MenuItemCompat.getActionView()` call in `MainActivity#42` (`getActionView()` is deprecated).

**Comment by boswelja** [comment]
This appears fixed in my pull request

File: library/src/main/java/com/afollestad/aesthetic/Util.java [hunk 1]
```diff
@@ -181,17 +178,10 @@ public static int adjustAlpha(
...
   static void setOverflowButtonColor(@NonNull final Toolbar toolbar, final @ColorInt int color) {
-    @SuppressLint("PrivateResource")
-    final String overflowDescription =
-        toolbar.getResources().getString(R.string.abc_action_menu_overflow_description);
-    final ArrayList<View> outViews = new ArrayList<>();
-    toolbar.findViewsWithText(
-        outViews, overflowDescription, View.FIND_VIEWS_WITH_CONTENT_DESCRIPTION);
-    if (outViews.isEmpty()) {
-      return;
+    Drawable overflowDrawable = toolbar.getOverflowIcon();
```
Comment (boswelja) [inline library/src/main/java/com/afollestad/aesthetic/Util.java]: This could be final.

**Pull request merged**